### ISSUE TITLE
5.4.1 Sync to 5.5.2. Author App is now blank

### ISSUE THREAD

**Issue by Lucero7919** [OfficeDev/microsoft-teams-apps-company-communicator#1500]
Hi all, I was getting the 
"error occurred while trying to access this service" so I Sync'd the repo through the deployment center, and that came back successfully, I then restarted the app service. Since then, the Author page in Teams is completely blank.

Literally no error, nothing. No idea where to start looking to solve this.

Published version in Teams Admin Center still shows 5.4.1

**Comment by tiwariShiv7** [comment]
HI @Lucero7919, Thanks for reaching out. Please Follow this Auth Issue Fix (https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/Authorization-issue-fix) guide. Ideally it will solve this issue.

**Comment by Lucero7919** [comment]
No joy I'm afraid. Going to Teams in the browser still shows the "Sorry, an error occurred while trying to access this service." but Teams Client shows a blank page.

Here is the Deployment Center log
<img width="746" alt="image" src="https://github.com/user-attachments/assets/1406557d-c384-4861-8379-254196eb51a2">

Application Insiights is angry too:
<img width="1028" alt="image" src="https://github.com/user-attachments/assets/8051ec1a-57a8-4f2a-b125-d22ed83bd83e">

Teams AC still shows:
<img width="587" alt="image" src="https://github.com/user-attachments/assets/6ccc01b5-1dff-4725-b34b-f8726fa7efd2">

**Comment by v-jaygupta** [comment]
Could you please confirm, if the sync has been performed for all the resources i.e. for the 3 function apps and the app service ?
If not, please perform sync for all the above mentioned resources. Also, please follow the migration guide to perform the [migration steps](https://github.com/OfficeDev/microsoft-teams-apps-company-communicator/wiki/v5-migration-guide#upgrading-from-v5x-to-latest-version) mentioned under section  - **Upgrading from v5.x to latest version**.

**Comment by v-jaygupta** [comment]
@Lucero7919 Could you please update us on the status of the issue ?

**Comment by Lucero7919** [comment]
I had to redeploy from scratch and that was not without issue. I had to alter the script to drop the min azcli version to 2.36 and then it would only deploy through a VSCode PoSH terminal.

**Comment by Lucero7919** [comment]
@v-jaygupta, hopefully you will take this feedback as intended, to help you improve the deployment mechanism.

1. Downloading the GIT Repo amending the parameters and running deploy.ps1 DOES NOT WORK
2. The check for Azure CLI looks for the x86 variant, not the x64 variant. If you have the correct version of AzCLI installed (x64) your check still comes back saying it is not installed. 
3. AzCLI 2.46 does not work. There are cmdlets within the script which are deprecated or not in 2.46. I had to alter deploy.ps1 to 2.36 before it worked.
4. Using PowerShell or Terminal continually failed. Their PSVersion is 5.1.22621.3880. I ran it in VScode without error. Guess what, it's PSVersion is 5.1.22621.3880 (the same)
5. The data function app was empty, despite the PoSH deployment, so I had to configure and sync that with the Repo.

So yes, I finally managed to get it deploy.
